packages/react/src/components/Authenticator/shared/FormFields.tsx

```diff
@@ -5,9 +5,10 @@ import { useAuthenticator } from '@aws-amplify/ui-react-core';
 
 export function FormFields(): JSX.Element {
   const { fields } = useAuthenticator(({ route }) => [route]);
+  const removedQRFields = fields.filter((field) => field.name !== 'QR');
```

Comment:
Since we add in QR fields, we need to remove it back out again, so another field isn't created.

Reply:
reverted